README.md

```diff
@@ -1 +1,41 @@
-# lboard
\ No newline at end of file
+# lboard
+
+In this project, I have built a web app powered by the Leaderboard APIs. I've achieved by using the Webpack Framework.
+
```

Comment:
### Please note that this is the final step to complete this project. So, it's so important to leave it professionally and technically completed 🙏 As I see, it's technically perfect 😉 Now let's work on the professional side 🥰

- [ ] Please provide a screenshot of your app in the README file. It'll make it appear more professional and good-looking 😉

_You can always refer to the README file [template](https://github.com/microverseinc/readme-template) as a reminder._